requirements.txt

```diff
@@ -0,0 +1,3 @@
+datadog
```

Comment:
What's up with this diff?  Were these packages not already included?

Reply:
This happens to be what my clone produced

Reply:
That doesn't really answer the question.  If these packages are not necessary then you shouldn't add them to the requirements file because it will add unnecessary dependencies to this package.  If they are necessary, then i am curious why you need them when we didn't need them before.

Reply:
1. The datadog becomes necessary because of this line:
https://github.com/goodmanship/sqsworkers/blob/ac61ae4160cd23312ea35d9fae2625b9eb397986/tests/test_crew.py#L1
2. The pytest is necessary to run the unit tests
$ pytest
-bash: pytest: command not found
3. 'mock' is pulled in my pytest so it is not necessary
4. boto3 - if we are dealing with AWS SQS we would need boto3 IMHO. But let me know if there was a way we can do without it

Reply:
I am making this file as before.